source/MaterialXTest/GenGlsl.h

```diff
@@ -0,0 +1,55 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXTest/Catch/catch.hpp>
+
+#include <MaterialXGenGlsl/GlslShaderGenerator.h>
+#include <MaterialXGenGlsl/GlslSyntax.h>
+
+#include <MaterialXTest/GenShaderUtil.h>
+
+namespace mx = MaterialX;
+
+// Base GLSL code generation tester
+class GlslShaderGeneratorTester : public GenShaderUtil::ShaderGeneratorTester
+{
+public:
+    using ParentClass = GenShaderUtil::ShaderGeneratorTester;
+
+    GlslShaderGeneratorTester(const mx::FilePath& testRootPath, const mx::FilePath& libSearchPath,
+                                const mx::FileSearchPath& srcSearchPath, const mx::FilePath& logFilePath) 
+        : GenShaderUtil::ShaderGeneratorTester(testRootPath, libSearchPath, srcSearchPath, logFilePath)
```

Comment:
Minor: I think by the style guidelines the colon is supposed to go at the end of the line above.